index.html

```diff
@@ -14,6 +14,17 @@
   <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/main.css">
 
+  <link rel="stylesheet" type="text/css" href="./css/addtohomescreen.css">
+  <script src="./js/addtohomescreen.min.js"></script>
```

Comment:
We don't really need this, chrome supports it nativity and it adds a lot of bloat / code to maintain